app/frontend/stylesheets/user_dashboard/_show.scss

```diff
@@ -0,0 +1,153 @@
+#user-dashboard {
+  height: 100vh;
+  margin: 0;
+
+  .log-out {
+    margin-left: 20px;
+    text-align: right;
+  }
+
+  .navbar {
+    margin-top: 25px;
+    height: 100px;
+  }
+
+  .logo {
+    padding-top: 15px;
+    filter: invert(100%);
+  }
```

Comment:
I needed to change the color of the logo. Should I put that in the _navbar.scss component instead?

Reply:
Ah so that's what the invert is for.
Not then you can add
```
.navbar .logo {
  filter: invert(100%)
}
```
inside your stylesheet yes.
The reste shouldn't be needed though